docs/nb/bgs_main_map.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    target_pixel_density = geometry.targ_hpmap(randoms)
</pre><p>you need to remove the map plotting from targ_hpmap</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahbigwood/S4Mock/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>